lib/commands/install.js

```diff
@@ -129,12 +129,25 @@ class InstallCommand extends Command {
         ctx.installPath = path.join(process.cwd(), 'versions', resolvedVersion); // eslint-disable-line require-atomic-updates
     }
 
-    casper() {
-        // Create a symlink to the theme from the current version
-        return symlinkSync(
-            path.join(process.cwd(), 'current', 'content', 'themes', 'casper'),
-            path.join(process.cwd(), 'content', 'themes', 'casper')
-        );
+    defaultTheme(ctx) {
+        const semver = require('semver');
+        const version = ctx.version;
+
+        // Starting with version 5.63.0, we are shipping an updated version of Casper with breaking changes under a new name
+        // If installing a version < 5.63.0, we need to symlink Casper
+        // We don't want the theme to outpace Ghost (e.g. try to use features that don't exist in Ghost yet)
+        if (semver.gte(version, '5.63.0')) {
```

Comment:
Is Casper going to be fully removed (from the bundle) in newer versions of Ghost? If so, do you need to add a task/migration to _remove_ the Casper symlink? If the theme listing service and gscan are updated to be more resilient to invalid symlinks, I guess this wouldn't be an issue.